source/_components/cover.rflink.markdown

```diff
@@ -124,8 +123,49 @@ devices:
           description: The `aliases` which do not respond to group commands.
           required: false
           type: [list, string]
+        type:
+          description: Set to `standard` or `inverted`. `inverted` will invert the on/off commands sent to the RFLink device, `standard` will not invert the on/off commands sent to the RFLink device. When omitted, the commands to the RFLink device will be inverted if the ID of the device starts with 'newkaku'.
```

Comment:
I think that the description could be shorter

Reply:
I agree, too many words in the description field. Am I allowed to refer to a configuration example in another section to elaborate on the type property?

Reply:
I shortened it and hope it is still proper English.

Reply:
```suggestion
          description: The option to invert (`inverted`) on/off commands sent to the RFLink device or not (`standerd`).
```

This is a suggestion. If I understand correctly, the code will automatically invert when you start using the KAKU device? That is something important to mention elsewhere in the text 😉

Reply:
It will only invert if the device ID starts with 'newkaku', those are the ASUN-650 devices. I've mentioned it with the example I provided.

Reply:
I applied your suggestion.